src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Resources.resx

```diff
@@ -117,6 +117,15 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="ClassTemplateName" xml:space="preserve">
```

Comment:
@davkean and I talked about this yesterday. I'm replicating the way that JS is doing this. However, this will all become moot when we implement #136, as then we'll be using the file name of the template, which isn't localized.